docs/_includes/conditional-include-directive.adoc

```diff
@@ -0,0 +1,85 @@
+////
+== Conditional Include Directive
```

Comment:
The correct term is "conditional preprocessor directives". It comes from the C world. See https://gcc.gnu.org/onlinedocs/cpp/Conditionals.html. AsciiDoc supports a consolidated set, though there are plans to expand to add a few more conditions. See https://github.com/asciidoctor/asciidoctor/issues/514.